scripts/liveness

```diff
@@ -41,20 +41,20 @@ demote() {
 }
 
 status() {
-	if [[ -f liveness-syncing ]]; then
+	if [ -f liveness-syncing ]; then
 		echo "syncing"
 		let R=$RANDOM%6
```

Comment:
This is also a problem (same as line 56).

`let` is not POSIX and `$RANDOM` is undefined in POSIX.

you could use something like awk to get a random number and be pretty sure to stay inside the acceptable space of available tools.
Removing let is just a matter of converting it to standard arithmetic expression.

```suggestion
		R=$(awk 'BEGIN { srand(); print int(rand()*32768) }' /dev/null)
```

Reply:
Thank you, er, brother, the PR updated accordingly.